orga/tests/integration/components/routes/authenticated/campaign/assessment/list-test.js

```diff
@@ -235,4 +235,37 @@ module('Integration | Component | routes/authenticated/campaign/assessment/list'
       assert.dom('img[src="url-badge"]').exists();
     });
   });
+
+  module('when the campaign have stages', function() {
```

Comment:
`module('when the campaign has stages', function() {` : have -> has

Reply:
Done 👍